src/atmos_spectral/driver/solo/mixed_layer.F90

```diff
@@ -43,14 +43,21 @@ module mixed_layer_mod
 
 use   time_manager_mod, only: time_type
 
+#ifdef COLUMN_MODEL
+use    column_grid_mod, only: get_deg_lon, get_deg_lat
+use         column_mod, only: get_surf_geopotential
+use       spec_mpp_mod, only: grid_domain
+#else          
 use     transforms_mod, only: get_deg_lat, get_deg_lon, grid_domain
+! mj know about surface topography
```

Comment:
I'd remove/change the personalised comments if we can whilst we're modifying the file. There's another below.

Reply:
I'd prefer it if all of the commits here are related to the column model p/r, or are required for the model to compile / run more generally. If we want to tidy the comments in the fortran we can do so in a different p/r?

Reply:
Entirely up to you, I'm not going to force it, just it's something that's been mentioned in the past and I was made to do it for a previous PR. But agreed it's not directly relevant to this.

Reply:
Ok, I don't think I'll do it for this (and neither do I think this should be requested in the future for other p/rs). For e.g., there are a load of other `!mj`s in that file, and I'm not going to go through deleting all of them when they don't have anything to do with this p/r!